app/views/ingredients/_form_fields.html.erb

```diff
@@ -1,14 +1,14 @@
-<td><%= f_ingredient.text_field :quantity, class: 'form-control input--small' %></td>
-<td>
-  <%= f_ingredient.select :measurement_unit,
+<div class='ingredients-col flex-sm'><%= f_ingredient.number_field :quantity, class: 'form-control', step: 'any' %></div>
```

Comment:
Making it a number field should still allow inputting directly without having to use the step. Using `step: any` also eliminates any annoying validation stuff like making you go to the next step. 

"It's true, I read about it in a ~magazine~ stack overflow post"

Reply:
Nice! I'll give this a go and see if it fixes the glitch. 🎉